### DISCUSSION TITLE
Hatsune Miku: Project Diva MegaMix

### DISCUSSION THREAD

**Discussion by Made0fB1naries** [skyline-emu/skyline#515] [Game Requests]
I've been playing hatsune miku titles since the first Game dropped on playstation 3 and playstation portable in 2009.

**Comment by PixelyIon** [comment]
Local weeb developer @bylaws is fairly interested in getting this up and running so you might just be in luck.

**Reply by Made0fB1naries** [reply]
Ohhhh nice 😂 and i give goodluck to the team of skyline emu👀

**Comment by FensBK** [comment]
I really want to play megamix on my S20 FE

**Comment by Yoomonika** [comment]
So excited!!

**Comment by hope3x** [comment]
Preach.
